app/models/container_project.rb

```diff
@@ -32,7 +32,8 @@ class ContainerProject < ApplicationRecord
   virtual_total :replicators_count, :container_replicators
   virtual_total :containers_count,  :container_definitions
   virtual_total :images_count,      :container_images
-
+  virtual_column :first_quota_name, :type => :string
+  virtual_column :first_limit_name, :type => :string
```

Comment:
I think you should add `:uses => "container_quota"` to both of these to avoid a n+1 when including on a `ContainerProject ` report